packages/components/src/border-control/border-control-dropdown/component.tsx

```diff
@@ -198,20 +198,22 @@ const BorderControlDropdown = (
 							/>
 						</HStack>
 					) : undefined }
-					<ColorPalette
-						className={ popoverContentClassName }
-						value={ color }
-						onChange={ onColorChange }
-						{ ...{ colors, disableCustomColors } }
-						__experimentalHasMultipleOrigins={
-							__experimentalHasMultipleOrigins
-						}
-						__experimentalIsRenderedInSidebar={
-							__experimentalIsRenderedInSidebar
-						}
-						clearable={ false }
-						enableAlpha={ enableAlpha }
-					/>
+					{ colors ? (
+						<ColorPalette
+							className={ popoverContentClassName }
+							value={ color }
+							onChange={ onColorChange }
+							{ ...{ colors, disableCustomColors } }
```

Comment:
There was a TS error from the `colors` prop:

```sh
Types of property 'colors' are incompatible.
Type 'Colors | undefined' is not assignable to type '(Color | MultipleColors)[]'.
Type 'undefined' is not assignable to type '(Color | MultipleColors)[]'.
```

`colors` can be undefined, and `ColorPalette` doesn't allow an undefined `colors` prop.

So I added a ternary to only render `ColorPalette` if `colors` is truthy.

Reply:
Reverted this